src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -1651,17 +1784,20 @@ bool processOnChangeCondition
         if (condValues != NULL) {
             /* Check if some of the attributes in the NotifyCondition values list are in the entity.
              * Note that in this case we do a query for all the attributes, not restricted to attrV */
-            ContextElementResponseVector allCerV;
+            ContextElementResponseVector rawCerV, allCerV;
```

Comment:
Ok, I'll do that.

Just curiosity... is the "style checker" script detect this kind of style-break?